torch/_inductor/compile_fx.py

```diff
@@ -799,17 +798,19 @@ def get_input_idxs_to_check(
     inputs: Union[List[torch.Tensor], Sequence[int]],
     static_input_idxs: Sequence[int],
 ) -> Sequence[int]:
-    def is_aligned(storage_offset, dtype):
+    def is_aligned(input):
```

Comment:
nit: this logic is duped in `utils.py`, can probably just share it there?

Reply:
thanks, I think you're right that the previous logic was incorrect